static/js/components/forms/EditProfileForm.js

```diff
@@ -21,7 +22,27 @@ const getInitialValues = (user: User) => ({
   name:          user.name,
   email:         user.email,
   legal_address: user.legal_address,
-  profile:       user.profile
+  profile:       {
+    ...user.profile,
+    // Should be null but React complains about null values in form fields. So we need to convert to
+    // string and then back to null on submit.
+    // $FlowFixMe
+    job_function: isNil(user.profile.job_function)
+      ? ""
+      : user.profile.job_function,
+    // $FlowFixMe
+    company_size: isNil(user.profile.company_size)
+      ? ""
+      : user.profile.company_size,
+    // $FlowFixMe
+    leadership_level: isNil(user.profile.leadership_level)
+      ? ""
+      : user.profile.leadership_level,
+    // $FlowFixMe
+    years_experience: isNil(user.profile.years_experience)
+      ? ""
+      : user.profile.years_experience
+  }
```

Comment:
Works great, but maybe a function could be used here to simplify things a bit, something like:
```javascript
const getValueOrEmptyString = (property: ?string): string => (isNil(property) ? "" : property)

const getInitialValues = (user: User) => ({
  name:          user.name,
  email:         user.email,
  legal_address: user.legal_address,
  profile:       {
    ...user.profile,
    // Should be null but React complains about null values in form fields. So we need to convert to
    // string and then back to null on submit.
    // $FlowFixMe
    job_function:     getValueOrEmptyString(user.profile.job_function),
    // $FlowFixMe
    company_size:     getValueOrEmptyString(user.profile.company_size),
    // $FlowFixMe
    leadership_level: getValueOrEmptyString(user.profile.leadership_level),
    // $FlowFixMe
    years_experience: getValueOrEmptyString(user.profile.years_experience)
  }
})
```

Reply:
Sounds good